app/src/main/res/layout/fragment_attendee.xml

```diff
@@ -407,6 +425,7 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_gravity="center"
+                android:enabled="false"
```

Comment:
why?

Reply:
Will get enabled once loading of custom forms are done